src/containers/ClusterDetail/index.jsx

```diff
@@ -65,14 +114,128 @@ export default class ClusterDetail extends Component {
         title: 'Date Created', dataIndex: 'created', key: 'created', width: '13%', render: getParseDate,
       },
       {
-        title: 'Operation', dataIndex: 'operation', key: 'operation', width: '6%', render: () => '...',
+        title: 'Operation', dataIndex: 'operation', key: 'operation', width: '6%', render: (text, node) => <Popover content={renderTableOperation(node)}>...</Popover>,
       },
     ];
     const rowSelection = {
       selectedRowKeys,
       onChange: this.onSelectChange,
     };
 
+    return (
+      <Table
+        rowSelection={rowSelection}
+        columns={columns}
+        dataSource={data}
+      />
+    );
+  }
+
+  renderViewNodeModal = () => {
+    const { showViewNodeModal, viewNode } = this.state;
+
+    if (showViewNodeModal) {
+      return (
+        <Modal
+          className={styles.viewNodeModal}
+          width={1128}
+          visible={showViewNodeModal}
+          hideHeader
+          hideFooter
+          onCancel={this.closeViewNodeModal}
+        >
+          <div className={classNames(styles.detail, styles.nodeDetail)}>
+            <div className={styles.detailBase}>
+              <div className={styles.detailBaseInfo}>
+                <strong>{viewNode.name}</strong>
+                <p>
+                  <span className="id">{viewNode.id}</span>
+                  <Status type="active" name="Active" />
+                </p>
+              </div>
+              <i className="icon icon-close" onClick={this.closeViewNodeModal}/>
+            </div>
+            <ul className={styles.detailExpand}>
```

Comment:
this `ul` block could be abstracted to a component

Reply:
same as before